src/FakeItEasy/Creation/ProxyOptions.cs

```diff
@@ -3,24 +3,24 @@ namespace FakeItEasy.Creation
     using System;
     using System.Collections.Generic;
     using System.Globalization;
+    using System.Linq.Expressions;
 #if FEATURE_NETCORE_REFLECTION
     using System.Reflection;
 #endif
-    using System.Reflection.Emit;
 
     internal class ProxyOptions : IProxyOptions
     {
         private readonly List<Type> additionalInterfacesToImplement = new List<Type>();
         private readonly List<Action<object>> proxyConfigurationActions = new List<Action<object>>();
-        private readonly List<CustomAttributeBuilder> additionalAttributes = new List<CustomAttributeBuilder>();
+        private readonly List<Expression<Func<Attribute>>> additionalAttributes = new List<Expression<Func<Attribute>>>();

```

Comment:
I see we've retained the "additional" here. It's only internal, so not crucial, but do we want to more closely match the language of the public API?